docker/cellranger/1.1.0/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Supply a valid download link and md5sum from "https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest"
+
+FROM ubuntu:20.04
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install curl -y && \
+    rm -r /var/lib/apt/lists/*
+
+WORKDIR /opt
+
+ARG cellranger_url
```

Comment:
OK, that wasn't my understanding, but I don't think it is important enough to hold this up.

Reply:
@adthrasher I haven't pushed a `cellranger:1.1.0` image. How do we want to handle which version of CellRanger to use? `1.0.0` uses cellranger `7.0.1`. The link we provide in the new Dockerfile only supplies whatever the latest version is. Manually push with `7.0.1` because that's what's been tested? Bump to latest?

Reply:
I agree with Andrew. I just moved these to the top and tried the docker build, and it worked fine for me.

Also, I don't think we want to be actually packaging cell ranger and distributing it via GitHub packages. @adthrasher have we looked at the licensing at all around this?

Reply:
Yeah, we should probably get their license reviewed. The package is set to internal for now. Even that may not technically be allowable based on the terms of the 10x license.